centaur/src/main/resources/standardTestCases/drs_usa_hca.test

```diff
@@ -1,50 +0,0 @@
-ignore: true # See https://broadworkbench.atlassian.net/browse/BW-467
-name: drs_usa_hca
-testFormat: WorkflowSuccess
-backends: ["papi-v2-usa"]
-skipDescribeEndpointValidation: true
-
-files {
-  workflow: drs_tests/drs_usa_hca.wdl
-  options-dir: "Error: BA-6546 The environment variable CROMWELL_BUILD_RESOURCES_DIRECTORY must be set/export pointing to a valid path such as '${YOUR_CROMWELL_DIR}/target/ci/resources'"
-  options-dir: ${?CROMWELL_BUILD_RESOURCES_DIRECTORY}
-  options: ${files.options-dir}/papi_v2_usa_compute_default.options.json
```

Comment:
Is this options json now orphaned along with the test's WDL and inputs json? If so then it should be deleted just like the others.

Reply:
Looks like that